### ISSUE TITLE
[FIRAnalyticsConnector registerAnalyticsListener:withOrigin:] Crash

### ISSUE THREAD

**Issue by linziyiwj** [firebase/firebase-ios-sdk#11761]
### Description

-[FIRAnalyticsConnector registerAnalyticsListener:withOrigin:] EXC_BAD_ACCESS

```
Crashed: APMAnalyticsQueue
0  libobjc.A.dylib                0x2820 objc_msgSend + 32
1  *                              0x29e5900 __62-[FIRAnalyticsConnector registerAnalyticsListener:withOrigin:]_block_invoke + 307040
2  *                              0x2acbfac +[APMAnalytics notifyEventListenersOnSerialQueue:] + 1250828
3  libdispatch.dylib              0x2320 _dispatch_call_block_and_release + 32
4  libdispatch.dylib              0x3eac _dispatch_client_callout + 20
5  libdispatch.dylib              0xb534 _dispatch_lane_serial_drain + 668
6  libdispatch.dylib              0xc0a4 _dispatch_lane_invoke + 384
7  libdispatch.dylib              0x16cdc _dispatch_workloop_worker_thread + 648
8  libsystem_pthread.dylib        0xddc _pthread_wqthread + 288
9  libsystem_pthread.dylib        0xb7c start_wqthread + 8
```

### Firebase SDK Version

10.4.0

### Xcode Version

14.3

### Installation Method

CocoaPods

### Firebase Product(s)

AB Testing, Analytics, Crashlytics, In-App Messaging, Messaging, Performance, Remote Config, Storage

### Targeted Platforms

iOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by rizafran** [comment]
Hi @linziyiwj, first off, could you provide the detailed steps on how you encountered the issue?

**Comment by linziyiwj** [comment]
> Hi @linziyiwj, first off, could you provide the detailed steps on how you encountered the issue?

This problem has not been reproduced in the development environment. Currently, this problem can only be seen in the Firebase console.

**Comment by paulb777** [comment]
@linziyiwj Thanks for the report. We see a related code problem that we're targeting a fix for the 10.16.0 release

**Comment by tsunghung** [comment]
Firebase 10.16.0 includes the fix for this. Please reopen this ticket if the issue still occurs in 10.16.0 or later.
